Release notes for Red Hat Developer Hub 1.4- Update Release Note Texts (#758)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/snip-bug-fix-rhidp-5120.adoc

[id="bug-fix-rhidp-5120"]

> = Fixed the techdoc text selection during reporting a techdoc issue

Reviewer: ```suggestion
= Resolved the issue with text selection when reporting a TechDocs issue
```

> Before this update, the feature to report a documentation (techdoc) issue failed. Therefore, when a user selected a text in a techdoc, a large icon appeared instead of a tooltip-button.

Reviewer: ```suggestion
Previously, the feature to report a documentation (TechDoc) issue failed. Therefore, when a user selected a text in a TechDoc, a large icon appeared instead of a tooltip button.
```

> With this update, users can select texts when reporting a documentation (techdoc) issue.

Reviewer: ```suggestion
With this update, users can select texts when reporting a documentation (TechDoc) issue.
```
